setup: fix mac os x toolkit local install

francisco-perez-sorrosal opened the pull request:
Without the versions added in the commit, the `make first-run` command failed

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/cohere-ai/cohere-toolkit?pullRequest=131) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/cohere-ai/cohere-toolkit?pullRequest=131) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cohere-ai/cohere-toolkit?pullRequest=131) it.</sub>

francisco-perez-sorrosal commented:
After these changes, I can exercise finally the toolkit successfully in my mac:

```
╰─❯ $ curl --location 'http://localhost:8000/chat-stream' --header 'User-Id: test-user' --header 'Content-Type: application/json' --data '{"message": "hey"}'


data: {"event": "stream-start", "data": {"is_finished": false, "generation_id": "41296b9b-7bc5-419a-b4ed-163bc0d4ef73", "conversation_id": null}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": "Hey"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " there"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": "!"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " How"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": "'s"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " it"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " going"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": "?"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " I"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " hope"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " you"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": "'re"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " having"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " a"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " fantastic"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " day"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": "!"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " Is"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " there"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " anything"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " you"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": "'d"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " like"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " to"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " talk"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " about"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " or"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " any"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " questions"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " you"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " have"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " in"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " mind"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": "?"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " I"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": "'m"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " here"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " to"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " help"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " and"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " provide"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " some"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " enjoyable"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " and"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " informative"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": " conversations"}}

data: {"event": "text-generation", "data": {"is_finished": false, "text": "."}}

data: {"event": "stream-end", "data": {"response_id": "75598b24-7307-499f-955f-696ffe0e92c7", "generation_id": "41296b9b-7bc5-419a-b4ed-163bc0d4ef73", "conversation_id": "ce219a20-42e4-4aec-a06c-28563d6712da", "text": "Hey there! How's it going? I hope you're having a fantastic day! Is there anything you'd like to talk about or any questions you have in mind? I'm here to help and provide some enjoyable and informative conversations.", "citations": [], "documents": [], "search_results": [], "search_queries": [], "tool_calls": [], "finish_reason": "COMPLETE"}}
```

scott-cohere commented:
@francisco-perez-sorrosal thanks for doing this! ✅

scott-cohere commented on docker-compose.yml:
```diff
@@ -71,7 +71,7 @@ services:
     restart: always
     ports:
-      - 4000:4000
+      - 4001:4000
```
ive seen the other dependency issues but this feels like it's because you had port 4000 occupied prior to running toolkit? can you just clear your ports before running this to see if that works?

francisco-perez-sorrosal replied:
Yeah, I checked with lsof last Friday, but I had nothing in 4000. I'll double-check again 🙃 just in case

francisco-perez-sorrosal replied:
I rebooted my laptop and rechecked the port and this time I found the nxd process (NoMachine) using port 4000 in my laptop (Don't know why didn't appeared first time). I'll revert the last commit.

```
sudo lsof -i TCP:4000
COMMAND PID USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
nxd     595   nx    3u  IPv4 0x8312dde2fbb0b831      0t0  TCP *:terabase (LISTEN)
nxd     595   nx    4u  IPv6 0x8312dde7cb9787e1      0t0  TCP *:terabase (LISTEN)
```

tianjing-li replied:
@francisco-perez-sorrosal thanks for making that change, It's approve but you'll have to sign your commits so we can merge the changes

francisco-perez-sorrosal commented:
Hey, sorry for the delay in doing this. Updated the main branch, rebased my commits, squashed them into a single one and signed it. Hope is what you need! Thanks!

yodamad commented:
I had to remove completly `psycopg2` on my side otherwise it fails
`psycopg2-binary` seems to be enough

tianjing-li commented:
@francisco-perez-sorrosal pytest is failing because you need to update the lock file. run `poetry lock`

francisco-perez-sorrosal commented:
Done! Just, updated poetry.lock
